### ISSUE TITLE
Improvement: Add crt post filter for better native res image quality

### ISSUE THREAD

**Issue by gizmo98** [gonetz/GLideN64#2780]
I experimented with libretro shaders and mupen64plus-next/gliden64 with nativres=1 and 320p output. Output results look interesting. 2D elements look better. Lowres textures and low polygon count get quiet good hidden. Blur caused from bad signal quality adds some edge smoothing and reduces dithering artifacts. Is there interest to add such a filter? I think i can use FXAA shader can as a template and add another post processing shader. 

![mk64-2](https://github.com/gonetz/GLideN64/assets/6412699/ddb29e54-99fb-424f-a39d-e8cf8480a61d)
![ridge racer-1](https://github.com/gonetz/GLideN64/assets/6412699/87843011-c673-4a8b-b15a-eccad91f184b)

Libretro forum thread examples:
https://forums.libretro.com/t/please-show-off-what-crt-shaders-can-do/19193/5427?u=gizmo98
https://forums.libretro.com/t/please-show-off-what-crt-shaders-can-do/19193/5416?u=gizmo98

Libretro thread:
https://forums.libretro.com/t/gizmo-slotmask-crt-shader/41594/5

**Comment by gizmo98** [comment]
More examples. Nativres=1 vs Nativres=1 with CRT shader:
![Bakuretsi-3 - noshader](https://github.com/gonetz/GLideN64/assets/6412699/21e8951a-5bad-4595-9041-77c243c14128)
![Bakuretsi-3](https://github.com/gonetz/GLideN64/assets/6412699/dc25d4b7-580b-4e08-8661-3a971bbf5a5d)

![mischief makers-2 - no shader](https://github.com/gonetz/GLideN64/assets/6412699/f816e741-d14e-4014-a061-c39548adc5e1)
![mischief makers-2](https://github.com/gonetz/GLideN64/assets/6412699/174d7c6b-0e55-48aa-8bbb-9ab647492f11)

![mischief makers-3-no shader](https://github.com/gonetz/GLideN64/assets/6412699/3c8b74eb-1359-4cc6-89bd-bb8c60aaa644)
![mischief makers-3](https://github.com/gonetz/GLideN64/assets/6412699/329b988e-5c31-4e67-a7d6-82178e0278bb)

![Nushi Tsuri - noshader](https://github.com/gonetz/GLideN64/assets/6412699/ac5c2bd1-7368-4067-b3de-3c4aacee065a)
![Nushi Tsuri](https://github.com/gonetz/GLideN64/assets/6412699/aa4e8183-8172-41ee-bf14-1d7674336b6f)

![Puyo3-no shader](https://github.com/gonetz/GLideN64/assets/6412699/3edba7a2-07eb-49ed-9ae9-011e3e292b00)
![Puyo3](https://github.com/gonetz/GLideN64/assets/6412699/98b4eb0b-7450-4c06-bf5a-b3fbeed32f6f)

![Puzzle League2 - no shader](https://github.com/gonetz/GLideN64/assets/6412699/f0959c42-2988-45a9-ac0d-4c381de5a01c)
![Puzzle League2](https://github.com/gonetz/GLideN64/assets/6412699/f636fc28-94fc-4123-b896-1bd00ae3c16f)

![Yoshis Story2-no shader](https://github.com/gonetz/GLideN64/assets/6412699/88d09822-dbc6-47b7-86c2-85a3ac8948f7)
![Yoshis Story2](https://github.com/gonetz/GLideN64/assets/6412699/02c58fd2-2867-4c95-ad13-6b7af94e4c6e)

**Comment by olivieryuyu** [comment]
nice

**Comment by gonetz** [comment]
> Is there interest to add such a filter? I think i can use FXAA shader can as a template and add another post processing shader.

Yes, the results look interesting. It indeed can be added as another post-processing filter, as FXAA.
